dlp/snippets/inspect_content.py

```diff
@@ -930,6 +930,80 @@ def callback(message):
 # [END dlp_inspect_bigquery]
 
 
+# [START dlp_inspect_image_all_infotypes]
+def inspect_image_file_all_infotypes(
+    project,
+    filename,
+    mime_type=None,
+    include_quote=True,
+):
+    """Uses the Data Loss Prevention API to analyze strings for protected data in image file.
+    Args:
+        project: The Google Cloud project id to use as a parent resource.
+        filename: The path to the file to inspect.
+        mime_type: The MIME type of the file. If not specified, the type is
```

Comment:
this could be dropped ... i know the node sample did it, but it's not necessary since the API takes a content type of IMAGE ... then you can drop all the mime code

Reply:
yeah, thank you for your comment,  (praise: really liked the suggested change), I have updated the code, please check. Thank you